docs/source/gbd2019_models/causes/afib/index.rst

```diff
@@ -203,45 +189,16 @@ For example, the *Definitions* table for *SIR* and *With-Condition and Free of C
      - State Name
      - Definition
    * - S
-     - Susceptible
-     - Susceptible to {cause name}
-   * - I
-     - Infected
-     - Infected with {cause name}
-   * - R
-     - Recovered
-     - Infected with {cause name}
-
-
-**With-Condition and Free of Condition Model**
-
-.. list-table:: State Definitions
-   :widths: 1, 5, 10
-   :header-rows: 1
+     - **S**\ usceptible to Atrial fibrillation and flutter 
+     - Simulant that has not been diagnosed with atrial fibrillation and flutter.
+   * - A
```

Comment:
"A" or "P" for this state?  Or "C", which is more DisMod-y...

Reply:
Oh thank you, I bolded the wrong letter. We are going with "A" for Atrial fibrillation and flutter as the prevalent state.